models/public/single-human-pose-estimation-0001/description/single-human-pose-estimation-0001.md

```diff
@@ -0,0 +1,35 @@
+# single human-pose-estimation-0001
+
+## Use Case and High-Level Description
+Single human pose estimation model based on https://arxiv.org/pdf/1906.04104.pdf.
+
+## Specification
+
+| Metric                                                        | Value                   |
+|---------------------------------------------------------------|-------------------------|
+| AP(coco orig)                                                 | 68                      |
+| GFlops                                                        | 60.125                  |
+| MParams                                                       | 33.165                  |
+| Source framework                                              | PyTorch\*               |
+
+
+## Inputs
```

Comment:
Because it public model, please describe I/O both for original PyTorch model and converted model. [Example](https://github.com/opencv/open_model_zoo/blob/develop/models/public/alexnet/alexnet.md)